recipes/dnapi/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: dnapi
+  version: "1.1"
+
+build:
+  number: 0
+  skip: True  # [py27]
+
+source:
+  fn: master.zip
+  url: https://github.com/jnktsj/DNApi/archive/master.zip
```

Comment:
You don't need to close it. Just push to this branch and it will be updated.
Otherwise this PR looks good! :)